definitions/npm/file-saver_v2.x.x/test_file-saver_v2.x.x.js

```diff
@@ -0,0 +1,58 @@
+// @flow
+
+import { describe, it } from "flow-typed-test";
+import saveAs from "file-saver";
+
+let blob = new Blob([""]);
```

Comment:
Can you switch the `"` to `'`? Clearly relevant to #2774 ...

Reply:
Done. See the config file would be useful ;)